squid_cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
we use #!/bin/bash in other scripts, let's be consistent

Reply:
```
01_install_requirements.sh:#!/usr/bin/env bash
02_configure_host.sh:#!/usr/bin/env bash
03_build_installer.sh:#!/usr/bin/env bash
05_create_install_config.sh:#!/usr/bin/env bash
06_create_cluster.sh:#!/usr/bin/env bash
assisted_deployment_cleanup.sh:#!/usr/bin/env bash
assisted_deployment.sh:#!/usr/bin/env bash
assisted_deployment_utils.sh:#!/usr/bin/env bash
cache_cleanup.sh:#!/usr/bin/env bash
generate_clouds_yaml.sh:#!/usr/bin/env bash
host_cleanup.sh:#!/usr/bin/env bash
ironic_cleanup.sh:#!/usr/bin/env bash
must_gather.sh:#!/usr/bin/env bash
podman_cleanup.sh:#!/usr/bin/env bash
registry_cleanup.sh:#!/usr/bin/env bash
squid_cleanup.sh:#!/usr/bin/env bash
workingdir_cleanup.sh:#!/usr/bin/env bash
```